docs/additional-examples/evm-swap.md

```diff
@@ -0,0 +1,663 @@
+# Performing a Cross-Chain Swap with PKPs and Lit Actions
+In this section, we will demonstrate how you can use PKPs and Lit Actions to perform a cross-chain swap. There are three parts to this flow:
+
+- Using Lit PKPs as escrow
+- Using Lit Actions to define swap conditions
+- Implementing Mint/Grant/Burn with SDK
+
+## 1) Using Lit PKPs as escrow
+
+[PKPs](../user-wallets/pkps/overview) are decentralized key pairs generated by the Lit Network. Using Lit Actions, we can program them to sign transactions under specific conditions, enabling escrow mechanisms. We’ll explore how to execute a cross-chain swap with this setup.
+
+### Example:
+Suppose Alice holds 100 token A on Base and Bob holds 100 token B on Chronicle and they both agree to a swap of,
+- 4 token A from Alice to Bob on Base
+- 8 token B from Bob to Alice on Chronicle
+
+We have an [example code](https://github.com/anshss/lit-evm-swap) for the below solution.
+
+### How does this work?
+1.	Mint a PKP: Start by minting a PKP on the Lit Network.
+2.	Assign a Lit Action: Grant a specific Lit Action to the PKP using the Mint/Grant/Burn (M/G/B) approach, allowing it to generate signatures according to the programmed conditions.
+3.	Burn the PKP: Finally, burn the PKP to ensure it can only be used within the confines of the Lit Action logic.
```

Comment:
In step 2 you already specify that the pkp will be minted with m/g/b so this feels repetitive

Reply:
ig it should be necessary to describe the flow once in the start before moving forward or there wouldn't be much context around what the user is doing till he reaches step 3

later in [step 3](https://github.com/anshss/docs/blob/evmswap/docs/additional-examples/evm-swap.md?plain=1#L333) is a repetition to highlight the working of the function

lmk your view